From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jadon Fowler <jadonflower@gmail.com>
Date: Fri, 17 Jun 2016 02:26:58 -0700
Subject: [PATCH] Fix infinite loop when saving chunks

Running `/save-all flush` would start an infinite loop that prints:
    ThreadedAnvilChunkStorage (world): All chunks are saved

diff --git a/src/main/java/net/minecraft/server/ChunkRegionLoader.java b/src/main/java/net/minecraft/server/ChunkRegionLoader.java
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/src/main/java/net/minecraft/server/ChunkRegionLoader.java
+++ b/src/main/java/net/minecraft/server/ChunkRegionLoader.java
@@ -0,0 +0,0 @@ public class ChunkRegionLoader implements IChunkLoader, IAsyncChunkSaver {
                 if (this.c()) {
                     continue;
                 }
+                break; // Paper - fix infinite loop when saving chunks
             }
         } finally {
             this.f = false;
--